client/my-sites/checkout/composite-checkout/types/backend/payment-method.ts

```diff
@@ -196,6 +203,8 @@ export function translateCheckoutPaymentMethodToWpcomPaymentMethod(
 			return { name: 'WPCOM_Billing_Ebanx_Redirect_Brazil_Tef' };
 		case 'netbanking':
 			return { name: 'WPCOM_Billing_Dlocal_Redirect_India_Netbanking' };
+		case 'id-wallet':
```

Comment:
As I'd like to get rid of these functions in favor of their already-existing counterparts in calypso, I think we should use the same keys as the existing ones. In #40049 you'll see that the key for this is `id_wallet`, with an underscore, so we should probably change all of these to use that key instead.